bylaws.md

```diff
@@ -153,7 +153,19 @@ All members must provide and monitor an email address which will serve as the so
 Emails from the board must be sent from an @texaslan.org (http://texaslan.org/) email account.
 Any communications through social media must follow an official announcement in email form.
 
-**Article X. Amendments**
+**Article X. Plan of Dissolution and Liquidation**
```

Comment:
IMO we ought to tack this on the end instead of changing existing article numbers